README.md

```diff
@@ -12,7 +13,7 @@ KaTeX is a fast, easy-to-use JavaScript library for TeX math rendering on the we
  * **Self contained:** KaTeX has no dependencies and can easily be bundled with your website resources.
  * **Server side rendering:** KaTeX produces the same output regardless of browser or environment, so you can pre-render expressions using Node.js and send them as plain HTML.
 
-KaTeX supports all major browsers, including Chrome, Safari, Firefox, Opera, Edge, and IE 9 - IE 11. More information can be found on the [list of supported commands](https://khan.github.io/KaTeX/function-support.html) and on the [wiki](https://github.com/khan/katex/wiki).
+KaTeX supports all major browsers, including Chrome, Safari, Firefox, Opera, Edge, and IE 9 - IE 11.
```

Comment:
While I agree that these were the wrong place to link to these, I do wonder whether we should offer some deep links into the documentation.  The supported functions list is quite useful, in particular.  Maybe add a mention under "print quality" (when comparing to TeX?)  "See a list of supported LaTeX functions"?

Reply:
@edemaine I've added the link to the list of supported commands.